ambari-server/src/main/resources/Ambari-DDL-MySQL-CREATE.sql

```diff
@@ -333,7 +333,7 @@ CREATE TABLE user_authentication (
   authentication_type VARCHAR(50) NOT NULL,
   authentication_key VARCHAR(2048),
   create_time TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  update_time TIMESTAMP DEFAULT 0,
+  update_time TIMESTAMP DEFAULT NOW(),
```

Comment:
some code (in future) may be comparing update_time with 0 to see if it was not initialized. In this case, Ambari with MySQL database will behave differently compared to using other databases. How about adding similar thing to other DDLs?